data/scenario-test.cfg

```diff
@@ -532,6 +532,71 @@ Xu, Xu, Qxu, Qxu, Ql, Ql, Ql, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Xu, Gg, Gg, Gg, Gg, Gg
             vision=15
         [/modify_unit]
     [/event]
+    [event]
+        name=attack_end
+        first_time_only=yes
+
+        [filter_attack]
+            special=chance_to_hit
+        [/filter_attack]
+
+        [message]
+            speaker=unit
+            message="I used a chance_to_hit attack specials."
+        [/message]
+    [/event]
```

Comment:
I notice there's no `[filter_attack] special=magical` case. Is that intentional?

Reply:
Oops i forget this, i will correct test. No another change plannef